lib/views/pages/on-site.html

```diff
@@ -1,3 +1,40 @@
 <% extend 'lib/views/layouts/default' %>
 
 <h2>On Site</h2>
```

Comment:
"At the Event" is a bit more meaningful than "on site". Can you please change the Route, template name and Headline.